### ISSUE TITLE
Controller dpad support

### ISSUE THREAD

**Issue by laceous** [Zamiell/isaac-mod-config-menu#11]
Is there a reason that the dpad is disabled?

I see this in the code:
```lua
local ignoreActionButtons = {Controller.BUTTON_A, Controller.BUTTON_B, Controller.BUTTON_X, Controller.BUTTON_Y, Controller.DPAD_LEFT, Controller.DPAD_RIGHT, Controller.DPAD_UP, Controller.DPAD_DOWN}
```

If I remove the dpad options from that table then it seems to work, but I could be missing something.

It looks like the dpad was added to the ignore list here: https://github.com/Zamiell/isaac-mod-config-menu/commit/82fa2d9a4c573c64d9a1fae0f49f3abffaa24e83

**Comment by Zamiell** [comment]
@piber20

**Comment by Zamiell** [comment]
looks like piber isn't responding, but if you've done extensive testing with the dpad enabled and everything works, then you can submit a PR to re-enable it.

**Comment by laceous** [comment]
I've only tested with a single xbox controller. I wouldn't call that extensive.

**Comment by Zamiell** [comment]
well, let me know if you can wrangle a few other controllers to test with, i dont have any, so for now we can leave it the way it is

**Comment by laceous** [comment]
I've tested with the following controllers on windows 10:

xbox one (bluetooth)
steam controller (wired)
hyperkin duke (wired)
switch pro (wired)
ps4 (wired)

Removing the DPAD options from ignoreActionButtons seems to let the d-pad work for all of these controllers.

The game seems to recognize all of these as xinput devices. Some on their own and some via steam controller settings (switch/ps4).

If I disable playstation support in steam settings then the controller still works, but the game recognizes it as a ps4 controller. Strangely, I don't have to modify the code to get the d-pad to work in this case.

I also noticed that DeadSeaScrollsMenu (which is what the Encyclopedia mod uses) supports the d-pad so I think it makes sense to support it here.

I'll put in a PR.
